src/test/frontend/replicasetdetail/replicaset_service_test.js

```diff
@@ -19,52 +19,16 @@ describe('Replica set service', () => {
   let service;
   /** @type {!md.$dialog} */
   let mdDialog;
-  /** @type {!angular.$q} */
-  let q;
-  /** @type {!angular.$httpBackend} */
-  let httpBackend;
 
   beforeEach(() => {
     angular.mock.module(replicaSetDetailModule.name);
 
-    angular.mock.inject((kdReplicaSetService, $mdDialog, $q, $httpBackend) => {
+    angular.mock.inject((kdReplicaSetService, $mdDialog) => {
       service = kdReplicaSetService;
       mdDialog = $mdDialog;
-      q = $q;
-      httpBackend = $httpBackend;
     });
   });
 
-  it('should show delete dialog and delete object', (doneFn) => {
```

Comment:
I did this already because of the new clang-formatter. Will try again but I doubt that it's an issue with npm.